src/bridge/mod.rs

```diff
@@ -31,6 +31,9 @@ pub use ui_commands::{send_ui, start_ui_command_handler, ParallelCommand, Serial
 const INTRO_MESSAGE_LUA: &str = include_str!("../../lua/intro.lua");
 const NEOVIM_REQUIRED_VERSION: &str = "0.9.2";
 
+#[cfg(target_os = "macos")]
+const TTY_STARTUP_DIRECTORY: &str = "$HOME";
```

Comment:
This should perhaps be named as `HOME_DIRECTORY`, the naming is confusing right now, since it's used when you are running from the GUI, not the TTY.

Furthermore, we already use the `dirs` crate, so this could be
https://docs.rs/dirs/latest/dirs/fn.home_dir.html

Reply:
good catch, I'll replace it to use the `dirs` crate instead